File: DS-Algo-with-Java/Lecture-13 Arrays/SortedOrNot.java
import java.util.Scanner;

public class SortedOrNot {

    static boolean isSorted(int arr[]) {
        boolean check = true;

        for (int i = 0; i < arr.length - 1; i++) {
            if (arr[i] > arr[i + 1]) {
                check = false;
            }
        }
        return check;
    }

    public static void main(String[] args) {
        
        try (Scanner sc = new Scanner(System.in)) {
            
            System.out.print("Enter size of array: ");
            int n = sc.nextInt();

            int arr[] = new int[n];
            System.out.println("Enter the elements: ");
            
            for (int i = 0; i < n; i++) {
                arr[i] = sc.nextInt();
            }

            System.out.println("Sorted: " + isSorted(arr));
        }
    }
}

File: DS-Algo-with-Python/Lecture-13 Arrays/SortedOrNot.py
from array import *
    
def isSorted(arr):
    check = True
    for i in range(len(arr) - 1):
        if arr[i] > arr[i + 1]:
            check = False
    return check

n = int(input("Enter size of array: "))

arr = array("i")
print("Enter the elements: ")
for i in range(n):
    arr.append(int(input()))

print(f"Sorted: {isSorted(arr)}")